src/amo/components/CollectionList/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import "~core/css/inc/mixins";
+@import "~photon-colors/photon-colors";
+@import "~ui/css/vars";
+
+.CollectionList {
+  padding: $padding-page;
+
+  @include respond-to(large) {
+    padding: $padding-page-l;
+  }
+
+  .Card-contents {
+    font-size: $font-size-m-smaller;
+  }
+}
+
+.CollectionList-wrapper {
+  @include respond-to(large) {
+    display: grid;
+    grid-auto-flow: column dense;
+    grid-gap: 0 $padding-page-l;
+    grid-template-columns: minmax(300px, 35%) 1fr;
+    margin: 10px 0 0;
+
+    .CollectionList-list.CardList {
```

Comment:
Maybe it should be the `Card-contents` that needs `background: $white`, currently it looks like this:

![screen shot 2018-06-26 at 14 40 16](https://user-images.githubusercontent.com/217628/41912671-505213f0-7950-11e8-91d0-76c30566b2a7.png)
![screen shot 2018-06-26 at 14 40 20](https://user-images.githubusercontent.com/217628/41912673-506faf32-7950-11e8-98f7-7b1c14059076.png)